# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxPlatform.cpp
# Commit: 425313809206
# Full Hash: 425313809206add668132f5653c0bba3a91e79d5
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2021-08-24 09:47:24
# Description:
#   Bug 1726562 - Remove option to disable software webrender for popups. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D123159
# ==============================================================================

diff -r 60b0cb3f37fe -r 425313809206 gfx/thebes/gfxPlatform.cpp
--- a/gfx/thebes/gfxPlatform.cpp	Tue Aug 24 03:29:21 2021 +0000
+++ b/gfx/thebes/gfxPlatform.cpp	Tue Aug 24 04:06:34 2021 +0000
@@ -1191,15 +1191,6 @@
 bool gfxPlatform::UseWebRender() { return gfx::gfxVars::UseWebRender(); }
 
 /* static */
-bool gfxPlatform::DoesFissionForceWebRender() {
-  // Because WebRender doesn't currently support all of the tests that Fission
-  // runs in CI, we only require WebRender for users who are enrolled in the
-  // the Fission experiment. This applies to both the control and the treatment
-  // groups, so they are as comparable as possible.
-  return FissionExperimentEnrolled();
-}
-
-/* static */
 bool gfxPlatform::UseRemoteCanvas() {
   return XRE_IsContentProcess() && gfx::gfxVars::RemoteCanvasEnabled();
 }